runtime/src/lib.rs

```diff
@@ -286,12 +286,6 @@ parameter_types! {
 	// The lazy deletion runs inside on_initialize.
 	pub DeletionWeightLimit: Weight = AVERAGE_ON_INITIALIZE_RATIO *
 		RuntimeBlockWeights::get().max_block;
-	// The weight needed for decoding the queue should be less or equal than a fifth
-	// of the overall weight dedicated to the lazy deletion.
-	pub DeletionQueueDepth: u32 = ((DeletionWeightLimit::get() / (
```

Comment:
parameters matches the recent Shiden runtime's parameters
https://github.com/AstarNetwork/Astar/pull/708/files#diff-80518c95a1f023b8f716bf975c7a6ff78d4c6cc85e4363fa108e95d3b3b3c1c3L590